app/lib/common/pages/medications/medication.dart

```diff
@@ -27,7 +31,9 @@ class MedicationPage extends StatelessWidget {
               loading: () => Center(child: CircularProgressIndicator()),
               loaded: (medication) => _buildMedicationsPage(
                 filterUserGuidelines(medication),
-                context,
+                isOkGuideline: medication.guidelines[0].warningLevel ==
```

Comment:
Why does this need to be passed into the constructor if we also pass the filtered user medication? Isn't the same information already available in the first argument we are passing?